File: Console Input & output/QuadraticEquation/QuadraticEquation.cs
using System;
//Problem 6. Quadratic Equation
//Write a program that reads the coefficients a, b and c of a quadratic equation ax2 + bx + c = 0 and solves it (prints its real roots).
class QuadraticEquation
{
    static void Main()
    {
        Console.WriteLine("Pleace enter the coefficients a,b and c of the quadratic equation");
        Console.WriteLine("Enter number a: ");
        double a = double.Parse(Console.ReadLine());
        Console.WriteLine("Enter number b: ");
        double b = double.Parse(Console.ReadLine());
        Console.WriteLine("Enter number c: ");
       double c = double.Parse(Console.ReadLine());

        double D = (b * b - 4 * a * c);
        if (D <= 0)
        {
            Console.WriteLine("There is no real roots of the equation");
        }
        else if (D == 0)
        {
            Console.WriteLine("There is only one real root of the equation");
            double x1 = -b / (2 * a);
            Console.WriteLine("The root of the equation x = {0}", x1);
        }
        else
        {
            double discriminant = Math.Sqrt(D);
            double rootOne = (-b - discriminant) / (2 * a);
            double rootTwo = (-b + discriminant) / (2 * a);
            Console.WriteLine("The first root of the quadratic equation x1 = {0} ", rootOne);
            Console.WriteLine("The first root of the quadratic equation x2 = {0} ", rootTwo);
        }

    }
}

File: Javascript Fundamentals/Conditional statements/js/QuadraticEquation.js
// Write a script that reads the coefficients a, b and c of a quadratic equation ax2 + bx + c = 0 and solves it (prints its real roots).
// Calculates and prints its real roots.

//var a = console.prompt("Enter a");
//var b = console.prompt("Enter b");
//var c = console.prompt("Enter c");

var a = 1;
var b = 1;
var c = 3;
if (a == 0) {
	console.log("This is not an quadric equation!");
}
else {
	var discriminant = (b * b) - (4 * a * c);
	if (discriminant > 0) {
		var x1 = (-b + Math.sqrt(discriminant)) / (2 * a);
		var x2 = (-b - Math.sqrt(discriminant)) / (2 * a);
		console.log("The equation roots are: \nX1:" + x1 +" \nX2: " + x2);
	}
	else if (discriminant == 0) {
		var x = -b / (2 * a);
		console.log("The equation has only one root: " + x);
	}
	else if (discriminant < 0) {
		console.log("The equation doesn't have solution!");
	}
}